src/pal/src/exception/signal.cpp

```diff
@@ -101,6 +101,17 @@ int g_signalPipe[2] = { 0, 0 };
 
 DWORD g_dwExternalSignalHandlerThreadId = 0;
 
+static DWORD64 GetPc(CONTEXT *context)
```

Comment:
We already have the same helper in seh-unwind.cpp. It would be nice to move it e.g. to context.h as an inline function so that we don't have it at two places.